src/actions/vstudio/_vstudio.lua

```diff
@@ -338,7 +338,7 @@
 		end
 
 		-- Then the system libraries, which come undecorated
-		local system = config.getlinks(cfg, "system", "fullpath")
+		local system = config.getlinks(cfg, "system", "name")
```

Comment:
If the project specified libdirs, then they wouldn't have to add path informatio to the link:

```lua
libdirs { "../../bin" }
links { "somelib" }
```

My take was that if the script went to the trouble of providing path information I should maintain it. Do you have a case where that falls over?